modules/ROOT/partials/configuration/paste_block_drop.adoc

```diff
@@ -10,9 +10,9 @@ endif::[]
 
 Due to browser limitations, it is not possible to filter content that is dragged and dropped into the editor. When `{pasteblockdropname}` is set to true dragging and dropping content into the editor will be disabled. This prevents the unfiltered content from being introduced. Copy and paste is still enabled.
 
-Default value : `+false+`
```

Comment:
Unrelated to the changes. This looks to be missing the *Type:* line. Probably something to check and fix any missing in a follow-up Jira